### PR TITLE
Remove apt settings from project

### PR DESCRIPTION
**Pull request by aneuwald-ctw**
**User-Facing Changes**
Users current have problems to run apt commands due to this configuration, that is not being used anymore in Lichtblick.

**Description**
Removed completely the functionality of update on ubuntu via APT, as we do not currently support this.

**Checklist**

- [x] The web version was tested and it is running ok
- [x] The desktop version was tested and it is running ok

### CODE REVIEW

File: packages/suite-desktop/resources/linux/deb/postinst [hunk 1]
```diff
@@ -1,46 +0,0 @@
...
-
-# Link the binary into global /usr/bin
-ln -sf '/opt/${productFilename}/${executable}' '/usr/bin/${executable}'
-
-# SUID chrome-sandbox for Electron 5+
-chmod 4755 '/opt/${productFilename}/chrome-sandbox' || true
-
-update-mime-database /usr/share/mime || true
-update-desktop-database /usr/share/applications || true
-
```
Comment (Woodii1998) [inline packages/suite-desktop/resources/linux/deb/postinst:15]: The lines of code above should be retained as they ensure that the software can be correctly recognized and launched in the browser on Linux.

Comment (Woodii1998) [reply packages/suite-desktop/resources/linux/deb/postinst:15]: In the Linux system, you can type lichtblick:// in the address bar of the browser to verify my statement. If you delete the above four lines of code, the browser will not be able to correctly trigger the application.

Comment (aneuwald-ctw) [reply packages/suite-desktop/resources/linux/deb/postinst:15]: Yes it makes sense. As I don't have Linux here to test at the moment, I will restore the files and remove only the APT part

**Review by laisspportugal — APPROVED** [review]
Looks good to me!

**Review by Woodii1998 — COMMENTED** [review]
LGTM

**Pull request merged**